.changeset/dry-shirts-refuse.md

```diff
@@ -0,0 +1,6 @@
+---
+"@khanacademy/math-input": patch
+"@khanacademy/perseus": patch
+---
+
+[LEMS-2880] Bugfix the close button focus outline on hover in keypad and other style issues
```

Comment:
Two things:
  * let's avoid JIRA ticket numbers in the changelog entries. This is an open project and JIRA is closed so not everyone will be able to reference the ticket. 
  * Regarding wording, what exactly was fixed with the outline? and what are the other styling issues?  Basically, if I read this in the changelog of the new release, how do I know what specifically was fixed? :)